samigo/samigo-services/src/java/org/sakaiproject/tool/assessment/services/assessment/AssessmentService.java

```diff
@@ -1101,6 +1101,7 @@ public SecurityAdvice isAllowed(String arg0, String arg1,
 						for (String attachment : attachments) {
 							String resourceIdOrig = "/" + StringUtils.substringAfter(attachment, "/access/content/");
 							String resourceId = URLDecoder.decode(resourceIdOrig);
+							resourceId = resourceId.trim();
```

Comment:
```suggestion
							String resourceId = StringUtils.trim(URLDecoder.decode(resourceIdOrig));
```

Reply:
StringUtils.trim adds the typical null checks

Reply:
I prefer to stick with the change as is. 

It has been tested and there are other parts on the same file that applies the same treatment to the resourceId, since it's correct I prefer to stick with tested code!

Thanks for the review